lib/components/atoms/Icon/Icon.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+/**
+ *
+ * Icon
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+
+import styles from './Icon.style';
+import type { Props } from './types';
+
+const Icon = ({ className, ...others }: Props): Node => {
+  return (
+    <div role="presentation" className={classnames('icon-wrapper', className)} {...others}>
+      <i className="icon" />
```

Comment:
We should have a prop that defines Icon name and it can be placed within the body of <i>Close<i> for accessibility